ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino

```diff
@@ -753,13 +754,12 @@ void SendCommands() {
     bool noResponsePending = commandQueue.isAckEmpty();
     if (noResponsePending || printerUsedBuffer < PRINTER_RX_BUFFER_SIZE * 3 / 4) {  // Let's use no more than 75% of printer RX buffer
       if (noResponsePending)
-        resetSerialReceiveTimeout();    // Receive timeout has to be reset only when sending a command and no pending response is expected
+        restartSerialTimeout(KEEPALIVE_INTERVAL);   // Receive timeout has to be reset only when sending a command and no pending response is expected
       Serial.println(command);          // Send to 3D Printer
       printerUsedBuffer += command.length();
       lastCommandSent = command;
       commandQueue.popSend();
-      if (command.startsWith("M109"))   // known commands that make the printer busy add time for timeout. This should be implemented with an array for all the commands like this.
```

Comment:
G28 (Home), M190 and M109 (wait for heating) have to wait longer for the ok, as I proposed, two timeout should do the trick. 
one for checking serial communication that restart on every char received (the waiting time is some msecs)
 another for waiting the "ok", that restart everytime you send a command (and the waiting time is variable msec to minutes)

Reply:
Theoretically when you send M109 no ok will be got until temp is reached. In between marlin will send busy every 2 seconds (this is what firmware does by default). Program will send M109 and start a 2.5 seconds timeout. Before it raise timeout error a busy should come reinitializing timeout to wait other 2.5 sec. No need for special long commands check should be required. No 2 timeout required then

Reply:
ok...maybe
`restartSerialTimeout(500);    // Once a char is received timeout may be shorter`
may be the problem...if you have autotemp report. It short timeout. I'll try to change it a little bit. What I want is not to add 500 every char is read but to be sure to have at least 500 (even it it will be correct a lower value in my opinion...100 should be more than enough)
Give me some minutes and I'll fix

Reply:
> Theoretically when you send M109 no ok will be got until temp is reached. In between marlin will send busy every 2 seconds (this is what firmware does by default). Program will send M109 and start a 2.5 seconds timeout. Before it raise timeout error a busy should come reinitializing timeout to wait other 2.5 sec. No need for special long commands check should be required. No 2 timeout required then

ok but with the current code is timing out, so not enough time or the reset is making it timeout. 

> ok...maybe
> `restartSerialTimeout(500); // Once a char is received timeout may be shorter`
> may be the problem...if you have autotemp report. It short timeout. I'll try to change it a little bit. What I want is not to add 500 every char is read but to be sure to have at least 500 (even it it will be correct a lower value in my opinion...100 should be more than enough)
> Give me some minutes and I'll fix

Yes, I put initialy 500 to get rid of too many timeouts, sounded like a good slack, it can be lower definetly.

Reply:
I posted a change...if you can try it it will be helpful